lib/acts_as_list.rb

```diff
@@ -1,4 +1,5 @@
 require 'acts_as_list/active_record/acts/list'
+require 'acts_as_list/active_record/acts/list/disabled'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.